docs/proposals/Drawer/README.md

```diff
@@ -0,0 +1,68 @@
+# Drawer
+
+The `<Drawer>` component will provide a dismissible right-hand side panel.
+
+## Design Patterns
+
+The `<Drawer>` component will allow users to view content that relates to their
+main task. It would appear on the right-hand side of the screen. It would allow
+the user to still view and interact with the main content. The user could
+dismiss it once they are finished.
+
+Some scenarios for a `<Drawer>` include the following:
+
+- Show a list of notes beside a job detail view
+- Show a work request beside a new quote form
+- Show a converted quote beside an active job
+- Show a list of insight cards beside a dashboard
+
+There are also some outstanding questions:
+
+- Should the drawer be full-height of the viewport or just the parent container?
+- How should the drawer animate when it opens and closed?
+- Should we include an option for left-hand and right-hand drawers?
+- Do all drawers need to be dismissible? What about an always-open version?
```

Comment:
If a drawer is always open I _think_ there's a good argument that it isn't a `Drawer` (because it's just a static column in a layout)... maybe we could keep with the container metaphor and call that a `Basket` 😆

Reply:
@chris-at-jobber It’s a drawer where something fell behind it and now it won’t close. 😉